src/content/basics/giant-swarm-operational-layers/index.md

```diff
@@ -0,0 +1,69 @@
+
+1. Infrastructure
+2. Giant Swarm Control Plane
+3. Giant Swarm API
+4. User Space
+
+### Infrastructure
+
+The infrastructre layer covers the area on top of actual (or virtual) machines, networking, etc., which is managed by Giant Swarm SREs usually accessed through VPN and bastion hosts as well as through the Cloud Provider APIs if applicable.
+
+This layer does not include the actual hardware and maintenance of the Data Center. This is either covered by the (internal or external) Data Center provider or by the Cloud provider.
+
+Giant Swarm SREs on this layer have root level SSH access to everything that pertains to a Giant Swarm installation. This is facilitated by a Single Sign On (SSO) mechanism including MFA. On the cloud they additionally have access to the Cloud account/subscription through a role to set up and manage the cloud resources.
+
+### Giant Swarm Control Plane
+
+The Giant Swarm Control Plane consists mainly of Services running inside the Control Plane Kubernetes cluster.
+
+Just like the former layer, this layer is accessed through VPN and bastion hosts. Giant Swarm SREs and Operations personel have cluster admin access to the Control Plane Kubernetes API through a tunnel, which is again facilitated by SSO with MFA.
+
+### Giant Swarm API
+
+The Giant Swarm API (GS API) is a customer facing API that is usually whitelisted for only a certain IP range within the customer's network. This layer covers the API itself, but also its client manifestations in form of the Happa Web UI and `gsctl` CLI.
+
+On this layer there's two levels of access:
+
+1. Giant Swarm API admin
+
+This access level is reserved for Giant Swarm Operations and Support personel and like the above layers facilitated by SSO with MFA.
+
+Such admin users have access to all organizations and all clusters in the Giant Swarm installation.
+
+2. Giant Swarm API user
+
+This is the standard type of GS API user that is given out to DevOps/Operations personel on the customer side. Usually that covers only few users that are tasked with cluster creation and management.
+
+Such users have access to all clusters in the organizations they belong to. They can be considered multi-cluster admins.
+
+### User Space
+
+The User Space layer is defined as the layer pertaining to a single Tenant Cluster Kubernetes API.
+
+Users on this level are either created by a Giant Swarm API user (in form of key pairs) or managed in an external Identity Provider (IdP), like [Azure AD](https://docs.giantswarm.io/guides/authenticating-with-microsoft-azure-active-directory/) or any other OIDC compliant IdP.
+
+However, a user with access to the Kubernetes API does not by default also gain any permissions, as the clusters are locked down by RBAC. Thus, a cluster admin first needs to create roles and bindings for the users. These roles can be defined as narrow or broad as needed for the specfic Tenant Kubernetes cluster. They can be bound to either single users or groups thereof.
```

Comment:
In general I would like to see more text wrapping. It is hard to comment on such blocks in general.